src/components/EmptyCart.jsx

```diff
@@ -0,0 +1,31 @@
+import { useNavigate } from "react-router-dom";
+import { Flex, Heading, Text, Button } from "@sparrowengg/twigs-react";
+const EmptyCart = () => {
```

Comment:
@arun-121 can you add a space between imports and function declaration.

Reply:
added space